src/Models/MagentoProduct.php

```diff
@@ -161,6 +161,7 @@ public function salePrice()
         }
         if ($saleEnd) {
             $saleEnd = new Carbon($saleEnd->value);
+            $saleEnd = $saleEnd->endOfDay();
```

Comment:
Uh, didnt know Carbon had an `endOfDay` method for 23:59:59

Reply:
Pretty wicked - they also have `->endOfWeek()`, minute.. hour.. etc. 💪

Reply:
I dig that, just looked at some docs and it comes with a `isEndOfDay` method too, that's awesome lol